lake_hylia.logic

```diff
@@ -1,3 +1,77 @@
 area LakeHylia:
-  room BeanstalkArea
-  room Main
+  room BeanstalkArea:
+    node EastArea:
+      door LeftLadder
+      door RightLadder
+    node WestArea:
+      door Ladder
+    node BeanstalkFusion:
+      door BeanstalkFusion
+    WestArea <-> BeanstalkFusion: flag HyliaBeanstalkFusion
+    node EastArea +: exit Virtual.Zeffa.FluteAway
+    node WestArea +: exit Virtual.Zeffa.FluteAway
+  room Main:
+    node Crest
+      entrance Crest
+      door MinishHouse
+      exit CrestHoleFusion
+      flag Virtual.Zeffa.LakeCrest
+    Crest -> NorthWest
+    node CrestHoleFusion:
+      entrance CrestHoleFusion
+    Crest -> CrestHoleFusion
+    node MinishWater:
+      door MinishSouth
+      door NorthHole
+      door EastVineHole
+    node MinishPortal:
+      door MinishPath
+    MinishPortal <-> MinishWater: item Flippers
+    MinishPortal -> SouthEast
+    SouthEast -> MinishPortal: item Boots
+    node DigCaveFusion:
+      door DigCave
+    Water <-> DigCaveFusion: flag HyliaDigCaveFusion
+    node IslandTop:
+      door MinishHouse
+      door Hole
+    IslandTop -> MinishWater: item Flippers
+    node SouthWest:
+      door East
+      door EastSouth
+      door South
+      enemy SouthWestChuChus
+    node Water:
+      chest SouthHP
+      door DropletsEntrance
+      enemy Octoroks
```

Comment:
The droplets entrance needs to be its own node (you can't reach the HP/enemies from that without cape/flippers)